sympy/core/tests/test_wester.py

```diff
@@ -75,7 +75,7 @@ def test_B2():
     d, e = FiniteSet(i), FiniteSet(j, k, l)
 
     assert (FiniteSet(i, j, j, k, k, k) & FiniteSet(l, k, j) &
-            FiniteSet(j, m, j)) == Union(a, Intersection(b, Union(c, Intersection(d, e))))
+            FiniteSet(j, m, j)) == Union(a, Intersection(b, Union(c, Intersection(d, FiniteSet(l)))))
     # {j} U Intersection({m}, {j, k} U Intersection({i}, {j, k, l}))
```

Comment:
this note should now read, `{j} U Intersection({m}, {j, k} U Intersection({i}, {l}))`:

```
>>> FiniteSet(i, j, j, k, k, k) & FiniteSet(l, k, j) & FiniteSet(j, m, j)
{j} U Intersection({m}, {j, k} U Intersection({i}, {l}))
```